server/common/deliverable/get.ts

```diff
@@ -0,0 +1,49 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { Sequelize } from "sequelize";
+
+import models from "db/models";
+
+import paginate from "helpers/paginate";
+import { resJsonMessage } from "helpers/res-json-message";
+
+interface propsWhere {
+  userId?: string | string[];
+  issueId?: string | number;
+}
+
+export default async function get(req: NextApiRequest, res: NextApiResponse) {
+  const { address, issueId } = req.query;
+  const where = {} as propsWhere;
+
+  if (address) {
+    const user = await models.user.findOne({
+      where: {
+        address: Sequelize.where(Sequelize.fn("LOWER", Sequelize.col("address")),
+                                 "=",
+                                 address),
+      },
+    });
+
+    if (!user) return res.status(404).json({ message: "user not found" });
+
+    where.userId = user.id;
+  }
+
+  if (issueId) {
```

Comment:
this should be done inside the findAndCountAll, if the issue does not exist then it will return an empty query, it's not a 404 because we are getting deliverables _and the route exists_ (404 = doesn't exist)